### PR TITLE
Add another variant to the new header test

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?

Based on some of the preliminary data analysis we have done we have decided to add another variant to the test.

This variant will have the 5 primary links go straight to a front.
image

And on each front of the main fronts for both variants we should see the corresponding menu as tertiary links. For example, on the main network front you would now see:

image

Feel free to chat to myself or @stephanfowler if you want to see some of the data!

Please note that no users will see this variant until a fastly change is made.
## What is the value of this and can you measure success?

We can gather even more data to help us make a decision about the new header
## Does this affect other platforms - Amp, Apps, etc?

Nope!
## Request for comment

@stephanfowler @zeftilldeath @SiAdcock

### CODE REVIEW

File: common/app/views/fragments/containers/facia_cards/container.scala.html [hunk 1]
```diff
@@ -87,18 +87,43 @@
...
-                                <li class="tertiary-navigation__item">
-                                    <a class="tertiary-navigation__link" href="@LinkTo(tertiaryLink.url)" data-link-name="nav2 : tertiary : @tertiaryLink.name">
-                                        <span class="tertiary-navigation__text">@Html(tertiaryLink.name)</span>
-                                        <i class="tertiary-navigation__icon rounded-icon"></i>
-                                    </a>
-                                </li>
+                            @* TODO: we should make sure that sport, culture, and life have tertiary links *@
+                            @* TODO: should this only happen on the second variant?*@
+                            @defining(Edition(request)) { edition: Edition =>
+                                @println(edition.networkFrontId)
```
Comment (SiAdcock) [inline common/app/views/fragments/containers/facia_cards/container.scala.html]: `println` still needed?

Comment (NataliaLKB) [inline common/app/views/fragments/containers/facia_cards/container.scala.html]: Nope! Thanks for that. I have removed it in https://github.com/guardian/frontend/pull/14676/commits/72c15a48fb61bb22840bcb9dd44fb537d26d97d6

File: common/app/mvt/MultiVariateTesting.scala [hunk 2]
```diff
@@ -27,6 +27,17 @@ object ABNewHeaderVariant extends TestDefinition(
   }
 }
 
+object ABNewHeaderVariantTwo extends TestDefinition(
+  name = "ab-new-header-variant",
```
Comment (SiAdcock) [inline common/app/mvt/MultiVariateTesting.scala]: Is it cool for this to take the same `name` as `ABNewHeaderVariant`?

**Comment by SiAdcock** [comment]
Love it, awesome variant 👍

**Comment by TBonnin** [comment]
Reverted there 91498652b9be1dad3a7c84247acc60e8946bfe0a. Reopening this PR

**Pull request merged**